frontend/src/visualization/filtering/Reset.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Button, Icon } from '@material-ui/core';
+import useService from '../../dependency-injection/useService';
+import FilterStateStore from '../../stores/filterState/FilterStateStore';
+import SchemaStore from '../../stores/SchemaStore';
+import FilterQueryStore from '../../stores/FilterQueryStore';
+import ShortestPathStateStore from '../../stores/shortest-path/ShortestPathStateStore';
+import EntityCountsStore from '../../stores/EntityCountsStore';
+
+/**
+ * Reset button used to reset all the stores that are used in the filter.
+ */
+export default function Reset(): JSX.Element {
+  const filterStateStore = useService(FilterStateStore);
+  const filterQueryStore = useService(FilterQueryStore);
+  const entityCountsStore = useService(EntityCountsStore);
+  const schemaStore = useService(SchemaStore);
+  const shortestPathStore = useService(ShortestPathStateStore);
+
+  const handleReset = () => {
+    filterStateStore.initFromSchema(schemaStore.getValue());
```

Comment:
Move this to a `init()` or `reset()` methon in the filter state-store and inject the schema store directly into the filter state-store. It's also possible that the `reset()` method of the filter state-store just iterates over all filter lines, setting the `active`flag to false and `clearing` the filter properties collection.